syntax/swift.vim

```diff
@@ -83,25 +83,31 @@ syntax keyword swiftKeywords
       \ if
       \ import
       \ in
+      \ indirect
       \ infix
       \ init
       \ inout
       \ internal
+      \ internal(set)
       \ is
       \ lazy
       \ let
       \ mutating
       \ nil
+      \ nonmutating
       \ operator
       \ optional
       \ override
       \ postfix
       \ prefix
       \ private
+      \ private(set)
```

Comment:
Yeah, seems they have no effect. I will remove.

![screen shot 2015-10-10 at 11 19 26](https://cloud.githubusercontent.com/assets/44164/10412634/cc72a98a-6f40-11e5-8679-07421f54cd05.png)

Added it because I saw it in others, also removed these.